include/zephyr/net/net_mgmt.h

```diff
@@ -76,6 +76,29 @@ struct net_if;
 
 /** @endcond */
 
+/** Central place the definition of the layer codes (7 bit value) */
```

Comment:
Should the codes be an `enum` so they are grouped nicely in the docs?

Reply:
We can certainly experiment with enum, the current doc does not look that great atm https://builds.zephyrproject.io/zephyr/pr/89083/docs/doxygen/html/group__net__mgmt.html
I will mark this as DNM and investigate this more.